lib/cli/input/init.js

```diff
@@ -0,0 +1,42 @@
+const inquirer = require('inquirer');
+
+const questions = [
```

Comment:
It is better to use domain-relevant entity names. This array can appropriately be called **credentials**.